#1.异常捕获

try{}catch(ex){}finally{}

>finally无论是否有异常，都会执行。即及时有异常发生走catch 代码块，最后还会执行finally 代码块。

#2.throw 抛出异常